SG Agreement.md

```diff
@@ -43,6 +43,14 @@ its interests, cast votes, and otherwise represent the interests of the [Steerin
 [Steering Group Member] may replace its [Steering Group Representative] at any time in its sole
 discretion.
 
+Each [Steering Group Member] may optionally appoint an individual
+"<a id="alternate-steering-group-representative">**Alternate Steering Group Representative**</a>". The
+[Steering Group Representative] may delegate any of their rights and responsibilities to the
+[Alternate Steering Group Representative]. In the absence of the Steering Group Representative,
+the Alternate Steering Group Representative may perform any of the actions that the Steering
+Group Representative can perform as defined above. In the event that both take action, the actions
+of the Steering Group Representative supersede.
```

Comment:
Discussing this internally we're a bit concerned this still leaves room for confusion. How about something like:

> The Steering Group representative and its corresponding Alternate Steering Group Representative cannot take action on the same subject simultaneously as this would get confusing as to whose voice counts. Only one can communicate on a given subject at any one time.